### DISCUSSION TITLE
Intel Arc GPU for decoding/re-encoding?

### DISCUSSION THREAD

**Discussion by RedAnon** [blakeblackshear/frigate#5108] [Ask A Question] [answered]
Since these have been available for quite some time. Does anyone use them not only for decoding the H264/H265 streams but also for re-encoding the streams to AV1?
Is that even possible with the current (or beta or git) Frigate version?

**Comment by NickM-27** [comment]
Encoding/ decoding av1 support for ark isn't coming until Linux kernel 6 which will be some time. Also I don't think any official ffmpeg builds support it yet

I believe others should work fine but not 10% sure

**Reply by RedAnon** [reply]
16? Really? Damn seeing that I'm currently on 6.1.6 I'll be probably dead by then.
Also I'm really glad you are only 10% not sure.

([Support for AV1 encoding on Arc HW has been in ffmpeg git for some time now.](https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/dc9e4789a3b504c08c8cd24e990aa692dde50bc6))

**Reply by NickM-27** [reply]
Sorry I had that wrong, meant linux kernel 6. 

> ([Support for AV1 encoding on Arc HW has been in ffmpeg git for some time now.](https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/dc9e4789a3b504c08c8cd24e990aa692dde50bc6))

Notice that I said "Also I don't think any official ffmpeg builds support it yet" because being committed and being included in a release are two different things. Meaning it may be usable but there may be bugs.

**Comment by emersonsc** [comment]
Any update on this?

**Comment by NickM-27** [comment, marked as answer]
A user has posted that they have arc working with frigate https://github.com/blakeblackshear/frigate/discussions/5108

as far as av1 goes, that requires a version off fmpeg compiled with `libvpl` which frigate stable does not have. Frigate dev (going to be 0.15) includes ffmpeg 7 which does support av1 encoding, you can also include your own ffmpeg binary if you want which would enable av1 as well.

**Reply by memarkham303** [reply]
I'm trying to get the Intel Arc GPU working in my Frigate installation.   Any idea how the user got it to work?
